index_V1

```diff
@@ -29,49 +69,141 @@
 		
 		<div class="grid-container">
 			
-			<div class="nav">
-				<ul>
-					<li><a href="index.html" style="color: skyblue;">ZEPHYRUS</a></li>
-					<li style="margin-right: -1vw;"><a href="login.php">Settings</a></li>
-					<li style="margin-right: 0vw;"><a href="login.php">Sign Out</a></li>
-				</ul>
-			</div>
+			<?php
+				require 'loading.php';
+			?>
 			
-			<div class="header"></div>
+			<div id="top" style="grid-row: 1;"></div>
+			
+			<div class="header" style="background-image: url('images/img_3.jpg'); background-size: 94vw 58vw; background-position: 28vw;">
+				
+				<div class="nav" style="z-index: 1; background-color:rgba(0, 0, 0, 0.2); position: sticky; top: 0vw;">
+					<ul>
+						
+						<li><a href="index.php" style="color: white;"><img src="images/icon_5.png" style="height: 1.95vw; width: 1.95vw; margin-right: 0.5vw; margin-bottom: -0.4vw;" draggable="false">ZEPHYRUS</a></li>
+						<?php if($User == "emmanueld" or $User == "EmmanuelD" or $User == "Emmanueld" or $User == "GaberahamL" or $User == "EmmanuelD" or $User == "EmmanuelD" or $User == "grahama") : ?>
+							<style>
+								.account:hover {
+									background: transparent;
+									transform: scale(1.02, 1.02);
+									transition: 0.6s;
+								}
```

Comment:
A bit of Login Error Catching Here. If the username is emmanuel, he is the admin and shall have a bonus link in the navigation called "settings"